src/options.json

```diff
@@ -1340,6 +1340,20 @@
           "namespace": "js"
         }
       },
+      "errors_only": {
+        "type": "boolean",
+        "default": false,
+        "description": "Fix errors but leave warnings (Supported by JS Beautify)",
+        "title": "Errors only",
+        "beautifiers": [
+          "JS Beautify"
```

Comment:
Looks like the docs need updating. `npm run docs`

Reply:
right now JS Beautify is set to use all javascript options. how do I tell it to use all except error_only?